### ISSUE TITLE
Possible to  option to have time in seconds rather than percentage

### ISSUE THREAD

**Issue by mapattacker** [plasma-umass/scalene#596]
**Problem**

Scalene has grown to be so easy to use, however there is one thing I still struggle with, i.e., the time is always displayed in %, rather than absolute time. That is why I still defer to line_profiler when profiling latency, which provide this feature.

**Proposal**

While there is an overall time given at the start, it will be so much more intuitive if there is an option to display both the time in seconds and/or percentage so that I do not need to make a mental calculation when reviewing at code or functional level.

**Comment by oostopitre** [comment]
Yes please understanding will improve with absolute time values too. Right now struggling to understand how these percentages sum to zero and what fraction of overall runtime do these percentages correspond to as well. Great library though. Very easy to use!
